titus_isolate/config/constants.py

```diff
@@ -23,7 +23,7 @@
 OVERSUBSCRIBE_FREQUENCY_KEY = 'TITUS_ISOLATE_OVERSUBSCRIBE_FREQUENCY'
 DEFAULT_OVERSUBSCRIBE_FREQUENCY = 60
 PREDICT_RESOURCE_USAGE_FREQUENCY_KEY = 'TITUS_ISOLATE_PREDICT_RESOURCE_USAGE_FREQUENCY'
-DEFAULT_PREDICT_RESOURCE_USAGE_FREQUENCY = 300
+DEFAULT_PREDICT_RESOURCE_USAGE_FREQUENCY = 300 # in seconds
```

Comment:
Actually let's default this to 0 for the moment.  So we have to consciously opt-in to turning it on.  This will be particularly helpful because we have to go delete old v1 resources before publishing these.